e2e/cypress/integration/plugins/systemconsole/drawPlugin_spec.js

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import 'cypress-file-upload';
+
+describe('Draw Plugin - Upload', () => {
+    after(() => {
+        // # Restore default configuration
+        const newSettings = {
+            PluginSettings: {
+                Enable: true,
+                EnableMarketplace: true,
+                MarketplaceUrl: 'https://api.integrations.mattermost.com',
+            },
+        };
+        cy.apiUpdateConfig(newSettings);
+    });
```

Comment:
Same here, only `Enable` and do config update at `before` hook.

Reply:
Save preferences or Update settings are not required on this test.Hence removed